doc/nrf/ug_multiprotocol_support.rst

```diff
@@ -0,0 +1,52 @@
+.. _ug_multiprotocol_support:
+
+Multiprotocol support for Thread and Zigbee
+###########################################
+
+The :ref:`ug_thread` and :ref:`ug_zigbee` protocols support running another protocol in parallel.
+
+This is achieved with the :ref:`nrfxlib:softdevice_controller`, an implementation of the :ref:`ug_ble_controller`.
+The SoftDevice Controller in turn relies on the functionalities of ref:`Multi-Protocol Service Layer (MPSL) <nrfxlib:mpsl>` library, which provides services for multiprotocol applications and allows the 802.15.4 radio driver to negotiate for transmission timeslots with the controller.
+For more information about how the SoftDevice Controller and MPSL work together, read the :ref:`SoftDevice Controller's documentation <softdevice_controller_readme>`.
+
+The multiprotocol solution available for Thread and Zigbee is dynamic.
+This means that it allows for running several radio protocols simultaneously without the time-expensive uninitialization and initialization in-between the switching.
+Switching between protocols requires only a reinitialization of the radio peripheral, since protocols may operate on different frequencies, modulations, etc.
+
+Enabling multiprotocol support
+******************************
+
+To enable the multiprotocol support for either Thread or Zigbee, set the following Kconfig option:
```

Comment:
@hubertmis Is it still valid statement that the new 802.15.4 radio driver will by default activate MPSL and enable multiprotocol support in that way?

Reply:
To enable multiprotocol support in the 802.15.4 driver MPSL must be enabled. Enabling SD controller only enables another protocol working in parallel. It is not necessary for multiprotocol applications. It's just an example.